README.md

```diff
@@ -67,6 +67,45 @@ The root directory where the file matching `fileInputPattern` will be searched f
 
 *Default:* `context.distDir`
 
+### jsonBlueprint
+
+The blueprint indicating what fields to read from the HTML file and convert into JSON.
+
+*Default:*
+```javascript
+base: {
+  selector: 'base',
+  attributes: ['href']
+},
+meta: {
+  selector: 'meta[name*="/config/environment"]',
+  attributes: ['name', 'content']
+},
+link: {
+  selector: 'link',
+  attributes: ['rel', 'href', 'integrity']
+},
+script: {
+  selector: 'script',
+  attributes: ['src', 'integrity'],
+  includeContent: false,
+}
+```
+
+### includeContent
```

Comment:
Might be better if this was a second level title as it's not strictly a config property, maybe like

```
## What if I want to include the content of a tag?
```

However, due to my incredibly tardy nature with this PR (and I'm sorry about that), I don't think we should hold this up any longer. We can fix this up later :)